packages/flutter_tools/lib/src/commands/build.dart

```diff
@@ -53,7 +53,7 @@ class _MaterialAsset {
     List<String> parts = name.split('/');
     String category = parts[0];
     String subtype = parts[1];
-    return '$category/$density/ic_${subtype}_${theme}_${size}dp.png';
+    return '$category/drawable-$density/ic_${subtype}_${theme}_${size}dp.png';
```

Comment:
What impact does this have on stocks' apk size? (It's fine either way, I just want to make sure we're aware of it.)